composer.json

```
@@ -27,7 +27,7 @@
         "php": "^8.1",
         "ext-mongodb": "^1.16",
         "composer-runtime-api": "^2.0",
-        "doctrine/mongodb-odm": "^2.6",
+        "doctrine/mongodb-odm": "^2.10@dev",
```

Update after the version is released.

```suggestion
        "doctrine/mongodb-odm": "^2.10",
```

---

I reverted the required update. The lazy ghost object option is enabled only when available.